homeassistant/components/aussie_broadband/config_flow.py

```diff
@@ -82,37 +77,23 @@ async def async_step_user(
             errors=errors,
         )
 
-    async def async_step_service(
-        self, user_input: dict[str, Any] | None = None
+    async def async_step_reauth(
+        self, user_input: dict[str, str] | None = None
     ) -> FlowResult:
-        """Handle the optional service selection step."""
-        if user_input is not None:
-            return self.async_create_entry(
-                title=self.data[CONF_USERNAME], data=self.data, options=user_input
-            )
+        """Handle reauth on credential failure."""
+        if user_input:
```

Comment:
We don't need this check now. The flow will always be started with the existing entry data as input.

Reply:
Done, Thanks.